docs/mkdocs.yml

```diff
@@ -321,6 +321,8 @@ nav:
         - retentionlabel list: cmd/purview/retentionlabel/retentionlabel-list.md
         - retentionlabel remove: cmd/purview/retentionlabel/retentionlabel-remove.md
         - retentionlabel set: cmd/purview/retentionlabel/retentionlabel-set.md
+      - retentioneventtype:
```

Comment:
Something went wrong with your merge I think, this section should be combined with the section on line 316